Kernel/CMakeLists.txt

```diff
@@ -276,7 +276,6 @@ set(KERNEL_SOURCES
     Net/Realtek/RTL8168NetworkAdapter.cpp
     Net/VirtIO/VirtIONetworkAdapter.cpp
     Net/IPv4/Socket.cpp
-    Net/LocalSocket.cpp
```

Comment:
More feng shui programming. We don't need a separate subdirectory for two files. The IP subdirectory (which you renamed recently to IPv4, and which we will rename back) will soon contain close to ten files, which is a level at which subdirectories actually make sense.